// af-collections/dist/index.js
"use strict";
/**
 *
 * Elijah Cobb
 * elijah@elijahcobb.com
 * https://elijahcobb.com
 *
 */
Object.defineProperty(exports, "__esModule", { value: true });
var AFArray_1 = require("./array/AFArray");
exports.Array = AFArray_1.AFArray;
var AFArrayList_1 = require("./array/AFArrayList");
exports.ArrayList = AFArrayList_1.AFArrayList;
var AFCSV_1 = require("./AFCSV");
exports.CSV = AFCSV_1.AFCSV;
var AFDictionary_1 = require("./map/AFDictionary");
exports.Dictionary = AFDictionary_1.AFDictionary;
var AFEnum_1 = require("./AFEnum");
exports.Enum = AFEnum_1.AFEnum;
var AFIterator_1 = require("./AFIterator");
exports.Iterator = AFIterator_1.AFIterator;
var AFMap_1 = require("./map/AFMap");
exports.Map = AFMap_1.AFMap;
var AFObject_1 = require("./AFObject");
exports.Object = AFObject_1.AFObject;
var AFQueue_1 = require("./AFQueue");
exports.Queue = AFQueue_1.AFQueue;
var AFSet_1 = require("./AFSet");
exports.Set = AFSet_1.AFSet;
var AFStack_1 = require("./AFStack");
exports.Stack = AFStack_1.AFStack;
